src/renderer/routes/test_sequencer_panel/components/AddWriteConditionalModal.tsx

```diff
@@ -2,28 +2,137 @@ import { Dialog, DialogContent } from "@/renderer/components/ui/dialog";
     >
       <DialogContent>
-        <Input type="text" value={value} onChange={handleInputChange} />
-        <Button onClick={() => handleWrite(value)}>Submit</Button>
+        <h2 className="text-lg font-bold text-accent1"> Add Conditional </h2>
+        <div className="flex">
+          <div>
+            <p className="text-muted-foreground"> Boolean Operation </p>
+            <div className="flex pt-2">
+              <Badge
+                variant={"outline"}
+                onClick={() => {
+                  addToValue("&");
+                }}
+                className="mr-2 flex-initial hover:cursor-pointer"
+              >
+                And
+              </Badge>
+              <Badge
+                variant={"outline"}
+                onClick={() => {
+                  addToValue("|");
+                }}
+                className="mr-2 flex-initial hover:cursor-pointer"
+              >
+                Or
+              </Badge>
+              <Badge
+                variant={"outline"}
+                onClick={() => {
+                  addToValue("!");
+                }}
+                className="mr-2 flex-initial hover:cursor-pointer"
+              >
+                Not
+              </Badge>
+              <Badge
+                variant={"outline"}
+                onClick={() => {
+                  addToValue("(");
+                }}
+                className="mr-2 flex-initial hover:cursor-pointer"
+              >
+                (
+              </Badge>
+              <Badge
+                variant={"outline"}
+                onClick={() => {
+                  addToValue(")");
+                }}
+                className="mr-2 flex-initial hover:cursor-pointer"
+              >
+                )
+              </Badge>
+            </div>
+          </div>
+          <div className="pl-10">
+            <p className="pb-1 text-muted-foreground"> Test Result </p>
+            <DropdownMenu>
+              <DropdownMenuTrigger className="inline-flex items-center rounded-md border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2">
+                Get Pass/Fail result of a test
+              </DropdownMenuTrigger>
+              <DropdownMenuContent>
+                <DropdownMenuLabel>Return the boolean</DropdownMenuLabel>
+                <DropdownMenuSeparator />
+                {tests.map((test) => {
+                  return (
+                    <DropdownMenuItem
+                      onClick={() => {
+                        addToValue(`$${test}`);
+                      }}
+                    >
+                      {test}
+                    </DropdownMenuItem>
+                  );
+                })}
+              </DropdownMenuContent>
+            </DropdownMenu>
+          </div>
+        </div>
+        <Input
+          type="text"
+          placeholder="$file.py::test_name & $file.py::test_name"
```

Comment:
this place holder confused me, maybe add "example: " as a prefix

Reply:
Or we could remove it since we adding the "Builder"